docs/source/installation.rst

```diff
@@ -1,5 +1,170 @@
-============
+=========================
 Installation
+=========================
+
+
+We define here two types of installation:
+
+- `Installation for standard users`_: for users who want to process data.
+
+- `Installation for contributors`_: for contributors who want to enrich the project (eg. add a new reader).
+
+We recommend users and contributors to use a `Virtual environment`_ to install GPM-API.
+
+
+Installation for standard users
+==================================
+
+pip-based installation
+..............................
+
+GPM-API is available from the `Python Packaging Index (PyPI) <https://pypi.org/>`__ as follow:
+
+
+.. code-block:: bash
+
+   pip install gpm_api
+
+
+Dependencies
 ============
 
-todo
+GPM-API depends on the ``cartopy`` Python package, which relies on the `GEOS <https://libgeos.org/>`_ library for its installation.
+
+Linux
+.....
+
+On Linux, prior to the installation of GPM-API, you can install GEOS using your package manager (e.g. ``apt install libgeos-dev``). Then, install ``gpm_api`` using ``pip`` (see above).
+
+Windows
+.......
+
+On Windows, the simplest method is to install Cartopy from the `conda-forge <https://conda-forge.org/>`_ channel using for example
+
+.. code-block:: bash
+
+   conda install cartopy>=0.20.0
+
+Alternatively, if you are not using ``conda`` as an environment manager, there are `unofficial already-build Cartopy wheels <https://www.lfd.uci.edu/~gohlke/pythonlibs/#cartopy>`_. Download the wheel corresponding to your system (see `naming scheme <https://docs.fileformat.com/compression/whl/>`_) and run
+
+.. code-block:: bash
+
+   pip install cartopy-0.xx.x-cp3x-cp3xm-win_xxxxx.whl
+
+
+
+
+Installation for contributors
+================================
+
+
+The latest GPM-API stable version is available on the GitHub repository `GPM-API <https://github.com/ghiggi/gpm_api>`_.
+
+Clone the repository from GitHub
+.........................................
+
+According to the `contributors guidelines <https://gpm-api.readthedocs.io/en/latest/contributors_guidelines.html>`__, you should first create a fork into your personal GitHub account.
+
+
+* Install a local copy of the forked repository:
+
+.. code-block:: bash
+
+    git clone https://github.com/<your-account>/gpm_api.git
+    cd gpm_api
+
+
+
+To install the project in editable mode :
+
+.. code-block:: bash
+
+	pip install -e .[image,dev]
```

Comment:
Does this work for you? I need to do a `pip install -e ".[image,dev]"` for local paths

Reply:
Oh, it does work for me without the quotes. I'll still add them